test/core/vite.config.ts

```diff
@@ -75,7 +75,7 @@ export default defineConfig({
     },
     server: {
       deps: {
-        external: ['tinyspy', /src\/external/, /esm\/esm/, /\.wasm$/],
+        external: ['tinyspy', /src\/external/, /esm\/esm/, /\.wasm$/, /src\/wasm-bindgen\//],
```

Comment:
There's something suspicious about this. I thought it only needs to externalize `index_bg.js`, but somehow Vitest got stuck unless `index.js` is also externalized.

There might be some issue with cyclic import, so I'll investigate it further.